socialdistribution/settings.py

```diff
@@ -128,4 +130,7 @@
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-AUTH_USER_MODEL = 'auth_provider.User'
+REST_FRAMEWORK = {
```

Comment:
In addition to this, I believe 'rest_framework' should be added to our INSTALLED_APPS above (it wouldn't let me comment above so I comment here) Source: the 'installation' section here https://pypi.org/project/djangorestframework/

Reply:
Good catch, thanks! Will update and merge